app/src/main/res/layout/item_speaker.xml

```diff
@@ -20,7 +20,7 @@
             android:layout_width="@{@dimen/search_speaker_image_size, default=@dimen/search_speaker_image_size}"
             android:layout_height="@{@dimen/search_speaker_image_size, default=@dimen/search_speaker_image_size}"
             android:layout_marginBottom="8dp"
-            android:layout_marginStart="8dp"
+            android:layout_marginStart="16dp"
             android:layout_marginTop="8dp"
```

Comment:
Sorry! I meant these marginTp and marginBottom should be 16dp :bow:

Reply:
😭

Reply:
oh..!
sorry I misunderstood😭